samples/NativeAOT/README.md

```diff
@@ -0,0 +1,89 @@
+# NativeAOT Sample
+
+This is the sample for using NativeAOT with .net Android
+
+## Debugging
+
+In order to debug we need to use `lldb`. First install the application
+
+```dotnetcli
+./dotnet-local.sh build samples/NativeAOT/NativeAOT.csproj -c Debug -t:Install
```

Comment:
Should we update this to recommend `Release` mode and `-p:DebugSymbols=true`?

Reply:
Personally, I prefer Debug, but ok 🤷